azavea/pfb-network-connectivity
# Change header links (#864)

## Overview

Methodology now links to a page on PFB's City Ratings site. Feedback
has been replaced with a link to the City Ratings home page.

## Testing Instructions

 * `./scripts/setup`
 * `vagrant ssh`
 * `./scripts/server`
 * See that Methodology now links to City Ratings methodology page
 * See that Feedback has been replaced with City Ratings, a link to City Ratings home page


## Checklist


Resolves #864
